### PR TITLE
Add standard headers

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by cloveras** [comment]
I linked from https://jira.vipps.io/browse/VLI-2237 to this PR.

File: docs/swagger.yaml [hunk 1]
```diff
@@ -497,3 +515,47 @@ components:
...
+      schema:
+        type: string
+        maxLength: 30
+        example: "Acme Commerce"
+    Vipps-System-Version:
+      name: Vipps-System-Version
+      in: header
+      description: |-
+        The version number of the solution.
+        See [http-headers](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/http-headers).
```
Comment (cloveras) [inline docs/swagger.yaml:536]:
```suggestion
        See [HTTP headers](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/http-headers).
```

File: docs/swagger.yaml [hunk 2]
```diff
@@ -497,3 +515,47 @@ components:
...
           parameter or is otherwise malformed.
+
+  parameters:
+    Vipps-System-Name:
+      name: Vipps-System-Name
+      in: header
+      description: |-
+        The name of the solution.
+        One word in lowercase letters is good.
+        See [http-headers](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/http-headers).
```
Comment (cloveras) [inline docs/swagger.yaml:526]:
```suggestion
        See [HTTP headers](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/http-headers).
```

File: docs/swagger.yaml [hunk 3]
```diff
@@ -497,3 +515,47 @@ components:
...
+      schema:
+        type: string
+        maxLength: 30
+        example: "acme-webshop"
+    Vipps-System-Plugin-Version:
+      name: Vipps-System-Plugin-Version
+      in: header
+      description: |-
+        The version number of the ecommerce plugin (if applicable).
+        See [http-headers](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/http-headers).
```
Comment (cloveras) [inline docs/swagger.yaml:557]:
```suggestion
        See [HTTP headers](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/http-headers).
```

**Review by tinolazreg — APPROVED** [review]
Looks good

**Review by tinolazreg — APPROVED** [review]
Looks good

**Pull request merged**